foundation-ui/utils/ReactUtil.js

```diff
@@ -15,6 +15,10 @@ const ReactUil = {
       return null;
     }
 
+    if (Array.isArray(elements) && elements.length === 1 && !alwaysWrap) {
```

Comment:
@mlunoe I decided against it so that strings, numbers, and `null` could still be returned without being wrapped. If we add the check, won't the same error occur when the non-renderable element is passed as children in the resulting call to `React.createElement`?